lightning/src/sign/ecdsa.rs

```diff
@@ -201,9 +226,9 @@ pub trait EcdsaChannelSigner: ChannelSigner {
 	/// Channel announcements also require a signature from each node's network key. Our node
 	/// signature is computed through [`NodeSigner::sign_gossip_message`].
 	///
-	/// Note that if this fails or is rejected, the channel will not be publicly announced and
-	/// our counterparty may (though likely will not) close the channel on us for violating the
-	/// protocol.
+	/// This method is *not* asynchronous, if this fails or is rejected, the channel will not be
```

Comment:
It might be just me but in several places in these docs, "fails or is rejected" feels like it could be clearer as "errors or panics." Fails/rejected seem almost like the same thing...